substrate/frame/revive/src/exec.rs

```diff
@@ -1285,7 +1303,7 @@ where
 		input_data: Vec<u8>,
 		allows_reentry: bool,
 		read_only: bool,
-	) -> ExecResult {
+	) -> Result<(), ExecError> {
```

Comment:
Should be the same?

Reply:
The returned output is now attached to the frame, so we can't return it anymore, or am I missing something?